Mixpanel/MixpanelPeople.m

```diff
@@ -32,7 +36,13 @@ - (NSString *)description
 
 - (NSDictionary *)collectAutomaticPeopleProperties
 {
-    NSMutableDictionary *p = [NSMutableDictionary dictionaryWithDictionary:@{@"$ios_version": [UIDevice currentDevice].systemVersion,
+    NSMutableDictionary *p = [NSMutableDictionary dictionaryWithDictionary:@{
+                                                                             @"$ios_version":
+#if defined(MIXPANEL_WATCH_EXTENSION)
```

Comment:
Seems weird that we have a file dedicated to collect watchOS properties for events but here we don't do it.